design/ceph/object/ceph-object-store.md

```diff
@@ -38,18 +38,15 @@ spec:
 
 Now create the object store.
 ```bash
-kubectl create -f object-store.yaml
+kubectl create -f ceph-object-store.yaml
 ```
 
 At this point the Rook operator recognizes that a new object store resource needs to be configured. The operator will create all of the resources to start the object store.
 1. Metadata pools are created (`.rgw.root`, `my-store.rgw.control`, `my-store.rgw.meta`, `my-store.rgw.log`, `my-store.rgw.buckets.index`)
-1. The data pool is created (`my-store.rgw.buckets.data`)
-1. A Ceph realm is created
-1. A Ceph zone group is created in the new realm
```

Comment:
For the simple case without multisite, the creation of realm, zone group, and zone still happen, right? Do we just need to clarify that these only get created if no zone is mentioned in the CR?